### DISCUSSION TITLE
How can I have a floating LSP error?

### DISCUSSION THREAD

**Discussion by AviAvinav** [neovim/neovim#24076] [Q&A] [answered]
This is a problem I have been dealing with for a while, I get an error but I can't view the full error, it's cut off.

<img src="https://i.ibb.co/xh8sQhH/Screenshot-2023-06-20-210211.png" />

Is there a way I can make the error into a floating window, sort of like how in the image below?

<img src="https://i.ibb.co/1b7x0C9/Screenshot-2023-06-20-210547.png" />

**Comment by clason** [comment, marked as answer]
`:h vim.diagnostic.open_float()`

**Reply by AviAvinav** [reply]
Thanks for resolving that!
